src/main/java/gui/NewMailDialog.java

```diff
@@ -33,7 +36,8 @@ public NewMailDialog(java.awt.Frame parent, boolean modal, Session session) {
      * regenerated by the Form Editor.
      */
     @SuppressWarnings("unchecked")
-    // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
+    // <editor-fold defaultstate="collapsed" desc="Generated
```

Comment:
This might be a problem, did you change anything here or it was formatted by the autoformatter?